packages/codec-components/src/react/components/codec/format.values.function-internal-raw-info-index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import type { Format } from "@truffle/codec";
+import { createCodecComponent } from "../../utils/create-codec-component";
+import { useInjectedNode } from "../../contexts/injected-node";
+import { Code } from "../common/code";
+
+export const { FunctionInternalRawInfoIndex } = createCodecComponent(
+  "FunctionInternalRawInfoIndex",
+  ({ functionIndex }: Format.Values.FunctionInternalRawInfoIndex) => {
+    const { prefix, content } = useInjectedNode();
+    return (
+      <Code>
+        {prefix?.prefix}(Function index: {functionIndex}){content?.suffix}
+      </Code>
```

Comment:
There aren't any injected nodes at the moment that would get here, so it's safe to remove them here.

Reply:
Oh, I see!  Yes as I'm sure you can tell I was doing this largely by copy and paste.  I don't actually know what the injected-node stuff means.  Mind explaining for the future? :)

Reply:
Ah so it's basically a way for more complex types to pass along contextual information to display data correctly. Think `ArrayValue` asking each element to add a `","` at the end. But yea in this case there isn't something like that.

Reply:
I see, makes sense, thank you!